### ISSUE TITLE
Install failing: Caused by fedoraproject.org moving to fedorainfracloud.org

### ISSUE THREAD

**Issue by simonluijk** [saltstack/salt-bootstrap#737]
For now I run this just after downloading install_salt.sh:

```
sed -i "s/http:\/\/copr.fedoraproject.org/https:\/\/copr.fedorainfracloud.org/" install_salt.sh
```

I have seen other PR's that seem to suggest embedding saltstack-salt-epel-6.repo so leaving this as an issue rather than a PR.

**Comment by pupapaik** [comment]
Same issue here. Can you fix it please. It broke up Kubernetes vagrant provisioning.

**Comment by mlalpho** [comment]
Fixed by #738 ?

**Comment by simonluijk** [comment]
Thank you.
